language-server/modules/langserver-core/src/test/resources/codeaction/create-variable/config-rename-positional-capability/createVariableInQueryAction3.json

```diff
@@ -20,7 +20,7 @@
               "character": 4
             }
           },
-          "newText": "xml<xml> xmlResult = "
+          "newText": "xml<xml:Element|xml:Comment|xml:ProcessingInstruction|xml:Text> xmlResult = "
```

Comment:
is this change correct?

Reply:
This is correct right. 
```ballerina
xml x1 = xml `<book>The Lost World</book>`;
    from xml element in x1 select element;
```

Each item in this iteration is of type `xml:Element|xml:Comment|xml:ProcessingInstruction|xml:Text`. IMO this is correct